mustang_codegen/lib/src/service_generator/screen_service_generator.dart

```diff
@@ -145,122 +128,111 @@ class ScreenServiceGenerator extends Generator {
         
       extension \$$serviceName on $serviceName {
         void updateState() {
-          $screenState screenState = MustangStore.get<$screenState>() ?? $screenState();
-          if (screenState.mounted) {
```

Comment:
all occurrences of `mounted` check is deleted in this file

Reply:
okay